schemas/context/experienceevent.schema.json

```diff
@@ -26,9 +26,10 @@
         },
         "xdm:timestamp": {
           "title": "Timestamp",
-          "type": "string",
-          "format": "date-time",
-          "description": "The time when the first event of the interaction occurred."
+          "type": "integer",
+          "description": "The timestamp when the first event of the touchpoint occurred. Milliseconds since midnight of January 1, 1970.",
```

Comment:
The description refers to "touchpoint" which is not used elsewhere in this schema. 

What's the time zone of reference for January 1st, 1970?

Reply:
Should be "Experience Event". 

The timestamp is nothing more then a unix timestamp in the form of millisecond which has a clear understanding amongst those that use it.  We can likely convert this to a string with the date-time format if that means the same thing.

Reply:
Yes, they are semantically equivalent, with a couple of advantages:
- `date-time` is built in in JSON Schema and used throughout XDM
- UNIX time has ambiguities around leap seconds

Reply:
@trieloff will the date-time support millisecond by using 
"time-hour ":" time-minute ":" time-second [time-secfrac]" mentioned here? https://tools.ietf.org/html/rfc3339